src/main/java/io/confluent/connect/elasticsearch/ElasticsearchSinkConnectorConfig.java

```diff
@@ -147,10 +148,28 @@
       + " mapping conflict or a field name containing illegal characters. Valid options are "
       + "'ignore', 'warn', and 'fail'.";
 
+  public static final String CONNECTION_SSL_CONFIG = "elastic.https";
+  private static final String CONNECTION_SSL_DOC = "Override to specify whether to use SSL "
+      + "connection in all cases. By default, the connector will use SSL when at "
+      + "least one URL begins with 'https:'. This config allows users to specify SSL even if "
+      + "host is specified without a protocol.";
+  public static final String CONNECTION_SSL_DEFAULT = "false";
+
+  public static final String CLIENT_AUTH_REQ_CONF = "elastic.https.client.auth.required";
+  private static final String CLIENT_AUTH_REQ_DOC = "Specify whether the client (connector) will "
+      + "be required to authenticate itself for SSL communication.";
+  public static final String CLIENT_AUTH_REQ_DEFAULT = "false";
```

Comment:
This is an example of a config that I think might be best not typed as a boolean but rather as an enumerated list of values (e.g., `required` and `requested`). With the boolean, it's not clear that setting this to false corresponds to `requested`. IIUC, what's really happening is that this controls whether to "Require client certificate authentication, or terminate the connection if no suitable client certificate is found". If this is `false`, it still will sent a client certificate if one is found that is trusted by the server's CA, right? Is that really clear?

Reply:
I made this match AK config for this now :+1: 

Originally I was shooting for consistency with our Confluent Rest Utils, which uses a boolean.